### ISSUE TITLE
Custom Props & component.

### ISSUE THREAD

**Issue by 91labs** [arunghosh/react-time-line#233]
THis is actually good library, that has been able to group together by date, it would have been great, if we could pass custom Component and props

**Comment by arunghosh** [comment]
Yes. That will be useful. Thank you for pointing out. Will take it up.
